docs/source/tutorials/one_class_segmentation_2d_unet.rst

```diff
@@ -24,137 +27,360 @@ Download dataset
 
-      .. code-block:: json
+        .. tabs::
+
+            .. tab:: JSON File
 
-         "multichannel": false
+                Usually around **Line Number 30** in the ``config.json`` is where you can update the ``multichannel`` subkey within the ``loader_parameters`` sub-dictionary.
 
-      .. note::
+                .. code-block:: json
 
-         The multichannel approach requires that for each subject, the image contrasts are co-registered. This implies that
-         a ground truth segmentation is aligned with all contrasts, for a given subject. In this tutorial, only one channel
-         will be used.
+                    "multichannel": false
+
+                .. note::
+
+                    The multichannel approach requires that for each subject, the image contrasts are co-registered. This implies that
+                    a ground truth segmentation is aligned with all contrasts, for a given subject. In this tutorial, only one channel
+                    will be used.
 
     - ``training_time:num_epochs``: the maximum number of epochs that will be run during training. Each epoch is composed
       of a training part and an evaluation part. It should be a strictly positive integer.
 
-      .. code-block:: json
+        .. tabs::
+
+            .. tab:: JSON File
+
+                Usually around **Line Number 29** in the ``config.json`` is where you can update the ``slice_axis`` subkey within the ``loader_parameters`` sub-dictionary.
+
+                .. code-block:: json
 
-         "num_epochs": 100
+                    "num_epochs": 100
 
 Train model
 -----------
 
     Once the configuration file is ready, run the training:
 
-    .. code-block:: bash
+    .. tabs::
 
-       ivadomed --train -c config.json --path-data path/to/bids/data --path-output path/to/output/directory
+        .. tab:: Command Line Interface
+
+            .. code-block:: bash
+
+               ivadomed --train -c config.json --path-data path/to/bids/data --path-output path/to/output/directory
 
     - We can pass other flags to execute different commands (training, testing, segmentation), see `Usage <../usage.html>`__.
 
 
     - ``--path-output``: Folder name that will contain the output files (e.g., trained model, predictions, results).
 
-      .. code-block:: bash
+        .. tabs::
+
+            .. tab:: Command Line Interface
 
-         --path-output path/to/output/directory
+                  .. code-block:: bash
+
+                     --path-output path/to/output/directory
 
     - ``--path-data``: Location of the dataset. As discussed in `Data <../data.html>`__, the dataset
       should conform to the BIDS standard. Modify the path so it points to the location of the downloaded dataset.
 
-      .. code-block:: bash
+        .. tabs::
+
+            .. tab:: Command Line Interface
 
-         --path-data path/to/bids/data
+                .. code-block:: bash
+
+                    --path-data path/to/bids/data
```

Comment:
This part (lines 351-373) is duplicated information from the block above (lines 345-349).

I think we want to have only two options here:
- CLI with  `--train`, `--path-data`, and `--path-output` flags
- CLI without the flags

Reply:
Great point, I didn't audit the actually content thoroughly enough, this is definitely more confusion than it needs to be. 

Streamlined further to this: 
![image](https://user-images.githubusercontent.com/6720068/149809635-6d2d8ce2-5de3-46e4-a0b8-de62c46ad98c.png)